tutorials/timeseries_01.ipynb

```diff
@@ -133,7 +118,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot_ts(ts_constant.interp_time(time=t))"
+    "ts_constant.interp_time(time=t).plot(marker='o')"
```

Comment:
```suggestion
    "ts_constant.plot(time=t, marker='o')"
```
Does this work?
The less interp in user code the better 😃

Reply:
Don't know why I missed that 😆